specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -776,22 +781,52 @@
         "name": {
           "description": "The name of the configuration resource.",
           "type": "string"
+        }
+      }
+    },
+    "ConfigDataProperties": {
+      "description": "Configuration data properties",
+      "type": "object",
+      "properties": {
+        "exclude": {
+          "description": "Exclude the resource from Advisor evaluations. Valid values: False (default) or True.",
+          "type": "boolean"
         },
-        "properties": {
-          "description": "The list of property name/value pairs.",
-          "type": "object",
-          "properties": {
-            "exclude": {
-              "description": "Exclude the resource from Advisor evaluations. Valid values: False (default) or True.",
-              "type": "boolean"
-            },
-            "low_cpu_threshold": {
-              "description": "Minimum percentage threshold for Advisor low CPU utilization evaluation. Valid only for subscriptions. Valid values: 5 (default), 10, 15 or 20.",
-              "type": "string"
-            }
-          },
-          "additionalProperties": {
-            "type": "object"
+        "low_cpu_threshold": {
+          "description": "Minimum percentage threshold for Advisor low CPU utilization evaluation. Valid only for subscriptions. Valid values: 5 (default), 10, 15 or 20.",
+          "type": "string"
+        },
+        "digests": {
+          "description": "Advisor digest configuration. Valid only for subscriptions",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DigestConfig"
+          }
+        }
+      }
+    },
+    "DigestConfig": {
+      "description": "Advisor Digest configuration entity",
+      "type": "object",
+      "properties": {
+        "actionGroupResourceId": {
+          "description": "Action group resource id used by digest.",
+          "type": "string"
+        },
+        "frequency": {
+          "description": "Frequency that digest will be triggered. Value must conform to ISO 8601 standard and must be greater than equal to 1 day and less than or equal to 30 days.",
+          "type": "string"
+        },
+        "categories": {
+          "description": "Categories to send digest for. If categories are not provided, then digest will be sent for all categories.",
+          "type": "array",
+          "items": {
+            "enum": ["HighAvailability",
+            "Security",
+            "Performance",
+            "Cost",
+            "OperationalExcellence"],
+            "type": "string"
```

Comment:
For enum, consider applying this https://github.com/stankovski/AutoRest/blob/master/Documentation/swagger-extensions.md#x-ms-enum